ert_gui/tools/plot/storage_client.py

```diff
@@ -9,8 +9,8 @@ def convert_response_to_float(resp):
     return [float(x) for x in string.split(",")]
 
 class StorageClient(object):
-    def __init__(self):
-        self._BASE_URI = "http://127.0.0.1:5000/" # Default flask server
+    def __init__(self, base_url=None):
+        self._BASE_URI = base_url or "http://127.0.0.1:5000/" # Default flask server
```

Comment:
Not a fan of the implicit bool conversion here :sweat_smile: Would a better idea be to have the default url in the function signature instead? 
`def __init__(self, base_url="http://127.0.0.1:5000/"):`

Reply:
Some like it, some don't I've gotten to experience. Not an issue having it in the signature instead.

Reply:
ps: the reason I didn't add in the signature (which I would have normally of course), is that I though the default value is more of a last resort fallback, or implementation detail. I didn't think it fit within the `StorageClient` at all. Gotta stop over thinking ;)